sub/source/docs/pilot-control-failsafe.rst

```diff
@@ -0,0 +1,28 @@
+.. _pilot-control-failsafe:
+
+======================
+Pilot Control Failsafe
+======================
+
+Sub provides a failsafe mechanism in case pilot manual control is lost. Pilot control in Sub is provided by either or both of two ways: via GCS MAVLink RC override messages( stimulated by GCS Joystick) or by the optional RC control link.
```

Comment:
```suggestion
Sub provides a failsafe mechanism in case pilot manual control is lost. Pilot control in Sub is provided by either or both of two ways: via GCS MAVLink RC override or Manual Control messages (stimulated by GCS Joystick) or by the optional RC control link.
```